README.md

```diff
@@ -84,7 +84,9 @@ If your application is running outside of Google Cloud Platform, such as locally
   2. Copy the key somewhere your application can access it. Be sure not to expose the key publicly.
   3. Set the environment variable `GOOGLE_APPLICATION_CREDENTIALS` to the full path to the key. The debug agent will automatically look for this environment variable.
 
-3. Alternatively, if you are running your application on a development machine or test environment where you are using the [`gcloud` command line tools][gcloud-sdk], and are logged using `gcloud beta auth application-default login`, you already have sufficient credentials, and a service account key is not required.
+If you are running your application on a development machine or test environment where you are using the [`gcloud` command line tools][gcloud-sdk], and are logged using `gcloud beta auth application-default login`, you already have sufficient credentials, and a service account key is not required.
+  
+Alternatively, you may set the `keyFilename` or `credentials` configuration field to the full path or contents to the key file, respectively. Setting either of these fields will override both of the above options. (See the [default configuration](config.js) for more details.)
```

Comment:
@ofrobots I don't think my changes are inconsistent with google-cloud - in both cases, config options take precedence over other ways to supply credentials.

@matthewloring Though there's no direct equivalent to the way the Node gcloud library handle config options, it seems that options specified in-code overrides anything other ways of supplying credentials consistently among the gcloud libraries in other languages (see [Java](https://github.com/GoogleCloudPlatform/google-cloud-java#specifying-a-project-id) and [Python](https://google-cloud-python.readthedocs.io/en/latest/google-cloud-auth.html#explicit-credentials).